packages/atomic/src/components/facets/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -133,8 +150,18 @@ export class AtomicDateFacet implements InitializableComponent, BaseFacetState {
   }
 
   public render() {
+    if (this.searchStatusState.hasError) {
+      return;
+    }
+
+    if (!this.searchStatusState.firstSearchExecuted) {
+      return (
+        <FacetPlaceholer numberOfValues={this.numberOfValues}></FacetPlaceholer>
+      );
+    }
```

Comment:
Since we use the State with additional controllers, no unfortunately. The code is very simple here and in my experience writing decorators can get very complex and hard to maintain (as it overrides lifecycle methods)